webrtc-stats.html

```diff
@@ -3937,6 +3938,14 @@ <h2>
                   For remote candidates, this property is not present.
                 </p>
               </dd>
+              <dt>
+                <dfn>tcpType</dfn> of type <span class="idlMemberType">DOMString</span>
+              </dt>
+              <dd>
+                <p>
+                  As defined in [[WEBRTC]].
```

Comment:
I'd make this "RTC candidate type, as defined {{RTCIceTcpCandidateType}} and used in {{RTCIceCandidate}}